### ISSUE TITLE
Icon

### ISSUE THREAD

**Issue by saltsoftdrink** [sosauce/CuteCalc#3]
Add material themed icon

**Comment by sosauce** [comment]
Where do you want me to add these icons ?

**Comment by sosauce** [comment]
Oh you mean the app icon. It isn't my main priority right now, but i'll try :)

**Comment by sosauce** [comment]
Ah I see, I never did that before, i'll stick with the current icon for now but change it ASAP

**Comment by KyuutePrincess** [comment]
I agree, although wouldnt that kinda ruin the apps purpose? Id prefer it to stand out with the pastel pink instead of some dark colours, maybe a toggle might be possible if implemented? A pastel pink app theme might also be nice.

**Comment by Der-Flip** [comment]
I think app designers and Devs should design their app however they like ... This is one of the reason I see material design as an absolute plague.

But I also think you shouldn't put label it as something that the app doesn't achieve. My suggestion: use a different wording like "material inspired". This signifies you don't adhere to material super strictly but incorporate elements.

Tl;dr: fully agree with kyuute; advertise as "material inspired"

**Comment by sosauce** [comment]
Heya everyone. I found Material Icons as a good idea when enabled, but couldn't figure out how they work yet. Im not a designer either, so I'm gonna stick with the pink app color for now. Hope y'all understand :)

**Comment by Der-Flip** [comment]
> Really thanks for that man, Yes you're right I think I'm forcing the developer and demotivating it by that

I'm not saying you force them. I agree with you if there's a material label on it, there should be material in there.

But I also agree that material completely removes all creativity in good UI/UX design removing uniqueness, especially since material isn't perfect at all either.

You have a valid point but so does the dev and kyuute. It's not Dev's top priority and that is a very fair decision on their end.

Now it's Dev's decision on how to handle the input and make decisions according to their preferences :)

**Comment by sosauce** [comment]
> > I think app designers and Devs should design their app however they like ... This is one of the reason I see material design as an absolute plague.
> > But I also think you shouldn't put label it as something that the app doesn't achieve. My suggestion: use a different wording like "material inspired". This signifies you don't adhere to material super strictly but incorporate elements.
> > Tl;dr: fully agree with kyuute; advertise as "material inspired"
> 
> Really thanks for that man, Yes you're right I think I'm forcing the developer and demotivating it by that I apologise for that And, again Thank You 😃

Nah man, this is what lets me know what everyone wants. If you didn't open this issue, i wouldn't know about dynamic Icons, Now i know !

**Comment by sosauce** [comment]
Ahah, thanks !! Anyways, im going in vacation. Updates are paused. 🫡

**Comment by sosauce** [comment]
![df](https://github.com/sosauce/CuteCalc/assets/98750531/2f6f2b6f-106a-4613-a3b0-a4ed5e3ad251)
 🙂❤️
